packages/docprovider/jest.config.js

```diff
@@ -3,5 +3,49 @@
  * Distributed under the terms of the Modified BSD License.
  */
 
-const func = require("@jupyterlab/testing/lib/jest-config");
-module.exports = func(__dirname);
+const jestJupyterLab = require('@jupyterlab/testutils/lib/jest-config');
```

Comment:
You can simplify this normally - look at: https://github.com/jupyterlab/extension-cookiecutter-ts/blob/4.0/%7B%7Bcookiecutter.python_name%7D%7D/jest.config.js